adoc/bp_chap_suma_backup.adoc

```diff
@@ -156,12 +175,12 @@ We recommend you run a manual database backup immediately after installation, or
 ====
 When [command]``smdba`` is run for the first time, or if you have changed the location of the backup, it will need to restart your database before performing the archive.
 This will result in a small amount of downtime.
-Note that regular database backups will not require any downtime.
+Regular database backups will not require any downtime.
 ====
 
 
 .Procedure: Performing a Manual Database Backup
-. Allocate permanent storage space for your backup. In this procedure, we will be using an NFS share located at [path]``/var/spacewalk/``.
+. Allocate permanent storage space for your backup. In this procedure, we will be using a directory located at [path]``/var/spacewalk/``.
```

Comment:
I'd remove the actor from this sentence (I'm cool with "you", less so with "we" ;)). "This example uses a directory ..."

Reply:
yes, much better!